packages/ERTP/src/amountMath.js

```diff
@@ -86,12 +86,7 @@ const helpers = {
 
 /**
  * @template {AmountValue} V
- * @type {(value: V) =>
- *  V extends NatValue ? 'nat' :
- *  V extends SetValue ? 'set' :
- *  V extends CopySetValue ? 'copySet' :
- *  V extends CopyBagValue ? 'copyBag' :
- *  never}
+ * @type {(value: V) => AssetKindForValue<V>}
```

Comment:
In general, an asset kind determines a value type, but not the other way around. We could have another assetkind that uses bigint values but uses mod p math or some such. The mapping is currently invertible, so I guess this is OK, but it gives me pause.